### PR TITLE
[Feat] Editing project to be published on npm

### PR DESCRIPTION
**Pull request by laisspportugal**
**User-Facing Changes**
N/A

**Description**
Editing project info on package.json to be published on NPM.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# @foxglove/eslint-plugin
+# @lichtblick/eslint-plugin
 
-[![npm package](https://img.shields.io/npm/v/@foxglove/eslint-plugin)](https://www.npmjs.com/package/@foxglove/eslint-plugin)
+[![npm package](https://img.shields.io/npm/v/@lichtblick/eslint-plugin)](https://www.npmjs.com/package/@lichtblick/eslint-plugin)
 
-Foxglove default eslint configuration & rules.
+Default eslint configuration & rules.
 
 Please err on the side of conservative changes to this repo - multiple Foxglove projects should adopt a change before making it a default.
```
Comment (aneuwald-ctw) [inline README.md:7]: Would not be better to rename all Foxglove to Lichtblick?

File: rules/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ## Rules
 
-The following rules are provided by `@foxglove/eslint-plugin`. You may need to enable the plugin by adding `@foxglove` in the `plugins` section of your eslint configuration.
```
Comment (aneuwald-ctw) [inline rules/README.md:3]: I would mantain this comment, to enable plugin adding "@lichtblick" in the eslint config.

**Review by aneuwald-ctw — APPROVED** [review]
Please check my questions, but if you think it is ok as it is, it is approved.

Also, please copy and adapt the rulesets we have on our main project to here.

File: README.md (hunk 1 above)
Comment (laisspportugal) [reply README.md:7]: Done

File: rules/README.md (hunk 2 above)
Comment (laisspportugal) [reply rules/README.md:3]: Done

**Comment by laisspportugal** [comment]
> Please check my questions, but if you think it is ok as it is, it is approved.
> 
> Also, please copy and adapt the rulesets we have on our main project to here.

Changes done. Regarding the rulesets it will be done afterwards.

**Pull request merged**